server/handlers/requestHandlers.go

```diff
@@ -58,10 +68,73 @@ func (h *Handlers) LoginRequest(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	expirationTime := time.Now().Add(time.Second * 20)
```

Comment:
Can you create a constant for this time at the top of this file called AUTO_LOGOUT_TIME, and then pass that constant instead of time.Second * 20?